packages/toolkit/src/view/pipeline/view-pipeline/PipelineContentViewer.tsx

```diff
@@ -61,6 +61,9 @@ export const PipelineContentViewer = ({
       if (pipeline?.permission.canEdit) {
         content = <PipelineSettings pipeline={pipeline} onUpdate={onUpdate} />;
       }
+      else{
+        content = <PipelinePlayground pipeline={pipeline} releases={releases} />;
+      }
```

Comment:
@thewbuk Since we will do the redirection, we just make it be null if user don't have the permission to edit but enter the setting tab